tools/install-unprivileged.sh

```diff
@@ -328,7 +328,7 @@ mkdir tmp
 cd tmp
 
 if [ "$DIST" = el7 ]; then
-	OSUTILS="$OSUTILS lzo squashfs-tools"
+	OSUTILS="$OSUTILS lzo"
```

Comment:
It was a good catch to see that this was being included here, but the problem is that this script needs to also support installing older versions of apptainer as well.   You could either leave this in for now and add a comment to remove it later, or you could test to see if the main install has already installed mksquashfs.

Then the line `mv tmp/usr/*bin/*squashfs utils/libexec` below shouldn't be removed but just made to be optional like the lines just above and below it.

Reply:
Leaving the install-unprivileged.sh script as-is